extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -270,4 +273,17 @@ private function getContributorsDetails( $contributors ) {
 			];
 		} , $contributors );
 	}
+
+	private function getHeroImageUrl() {
+		$heroImageUrl = '';
+		$heroImage = Title::newFromText( self::COMMUNITY_PAGE_HERO_IMAGE, NS_FILE );
+		if ( $heroImage instanceof Title && $heroImage->exists() ) {
+			$heroImageFile = wfFindFile( $heroImage );
+			if ( !empty( $heroImageFile ) ) {
```

Comment:
Minor: I'd consider preferring the more specific `$heroImageFile instanceof File` over `empty`.